.github/workflows/upgrade-python-requirements.yml

```diff
@@ -13,8 +13,8 @@ jobs:
    call-upgrade-python-requirements-workflow:
     with:
        branch: ${{ github.event.inputs.branch }}
-       team_reviewers: 'axim-aximprovements'
-       email_address: 'aximimprovements@axim.org'
+       team_reviewers: 'edly'
```

Comment:
`edly` is not a valid github team right now so I think it can just go to the e-mail address and this line can be commented for now.

@ziafazal if you want that team to exist and have you and others who are CCs and members of the `openedx` org added to it, let us know and we can create it.

Reply:
@feanil I have commented the team reviewer for now.